x-pack/plugins/security/server/plugin.test.ts

```diff
@@ -41,7 +41,9 @@ describe('Security Plugin', () => {
     mockClusterClient = elasticsearchServiceMock.createCustomClusterClient();
     mockCoreSetup.elasticsearch.legacy.createClient.mockReturnValue(mockClusterClient);
 
-    mockDependencies = { licensing: { license$: of({}) } } as PluginSetupDependencies;
+    mockDependencies = ({
+      licensing: { license$: of({}), featureUsage: { register: jest.fn() } },
+    } as unknown) as PluginSetupDependencies;
```

Comment:
You can use the licensing plugin's mocks in `x-pack/plugins/licensing/server/mocks.ts` instead. That would also avoid the `unknown` cast.